src/_scss/pages/search/awardTypeFilter.scss

```diff
@@ -29,6 +26,10 @@
         top: .25rem;
     }
 
+    .primaryAwardTypeOption label div, .secondaryAwardTypeOption label div {
```

Comment:
FYI (for the future), you should take advantage of Sass's nesting for more readable and easier to develop code:
```
.primaryAwardTypeOption, .secondaryAwardTypeOption {
    label {
        &:before {
            top: .25rem;
        }
        div {
            display: inline;
        }
    }
}
```

This makes it easier to read and think about.